trunner/tools/phoenix.py

```diff
@@ -22,6 +22,23 @@ def wait_for_dev(port, timeout=0):
             raise TimeoutError
 
 
+def wait_for_vid_pid(port_hint: str, timeout=0):
+    asleep = 0
+    port = None
+
+    while port is None:
```

Comment:
I thinking if it's worth changing it to `while not port,` for better readability, but then the loop would check if it's not just None, but if it's not every state that python considers falsy